change/@azure-msal-node-extensions-8fbfdb4b-83ee-4c5f-a036-594a7aeb4f06.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Bump typescript version to 4.9.5 #5750",
+  "packageName": "@azure/msal-node-extensions",
```

Comment:
Changefiles for browser, common and node?

Reply:
Beachball root scope for v3 is `["lib/*", "extensions/*"]`. That's why there are no dedicated files for the packages you listed above.
@hectormmg

Reply:
More likely this is happening because the v3 branch already has changefiles and beachball is configured to use dev as the root, not the v3 branch. We should likely update the beachball config in the v3 branch to use the v3 branch as root.

Reply:
Beachball doesn't require changefiles for browser, common, etc even after updating the root branch https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5780.

Reply:
That's because the [command](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/112abff69ebb68d4c8a88e86af6d056cf650e492/package.json#L63) specifically references the branch